src/aat/java/uk/gov/hmcts/ccd/userprofile/befta/HighLevelDataSetupApp.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.ccd.userprofile.befta;
+
+import uk.gov.hmcts.befta.dse.ccd.CcdEnvironment;
+import uk.gov.hmcts.befta.dse.ccd.DataLoaderToDefinitionStore;
+
+public class HighLevelDataSetupApp extends DataLoaderToDefinitionStore {
+
+    public HighLevelDataSetupApp(CcdEnvironment dataSetupEnvironment) {
+        super(dataSetupEnvironment);
+    }
+
+    public static void main(String[] args) throws Throwable {
+        main(HighLevelDataSetupApp.class, args);
+    }
+
+    @Override
+    protected boolean shouldTolerateDataSetupFailure() {
```

Comment:
If you don't override this method like this, any failures in data setup calls will fail the build. In CCD's case we are more tolerant to failures in importing test definition files. Services teams may or may not have or skip this method override if they use Befta Framework for this practice.